README.md

```diff
@@ -1,2 +1,91 @@
-# currency-converter
-A React app that brings information about multiple currency conversions
+![](https://img.shields.io/badge/Microverse-blueviolet) ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB) ![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
+
+# Currency Converter - Global Edition - React Application
+
+> A react web application for converting between different currencies from around the world, updating it's values daily and even referencing past-date's values
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- React
+- Redux
+- Linters for React
+
+## Website
+
+[Click here to access the website!](https://stirring-creponne-35b474.netlify.app/)
+
+## Information
+
+**Work In Progress**
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+This project fetches information from the [Currency-API]()
+
+**This project is for an exercise for the microverse program. Features may be added across exercise completions**
+
+## To run this repository locally, clone this repository and run this command on the directory's terminal:
+
+```
+npm install
+```
+All the necessary packages will be installed with the command above
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Authors
+
+👤 **Lucas Mullen**
+
+- GitHub: [@mullenlucas](https://github.com/mullenlucas)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lucas-mullen-447312119/)
+
+## 🤝 Contributing
+
+ - Readme Template taken from [Microverse Readme Template](https://github.com/microverseinc/readme-template)
+ - Linters for React taken from [Microverse Linters](https://github.com/microverseinc/linters-config)
+ 
+Contributions, issues, and feature requests are welcome!
+
+## Show your support !
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To all the collaborators for the Readme Template and the Linters
+- The Microverse Team
+- My coding-partners and fellow cohort members
+- Image for the header logo taken from [here](https://www.pngegg.com/en/png-zmlqe)
```

Comment:
- [ ] Kindly add credit for the design used as inspiration to the original designer of the design used. It's a major requirement that you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please add this credit below the ones you already have in the Acknowledgment section 👍